source/development_process/1_development.rst

```diff
@@ -168,25 +165,31 @@ Developers will spend most of their time in the feedback loop which is the
 development process. It begins by going to the issue tracker and picking an
 open issue. In general, it is best to consult the project lead on which issues
 are the highest priority. Issues should also document their dependencies on
-other issues and pull requests, but look out because sometimes they do not.
-Next, create a new branch. Give the branch a descriptive name which is easy to
+other issues and pull requests, but look out, because sometimes they do not.
+
+Create a new branch. Give the branch a descriptive name which is easy to
 tab complete and has the issue number it fixes. A common naming scheme among
 OSL projects is: ``<descriptive name>/<issue type>/<issue number>``. Having a
 descriptive name in front makes the branch easy to tab complete. Having the
 issue type makes it very clear whether the issue is a feature or a
 bug, and the issue number indicates what issue the branch belongs to in the
-issue tracker. Developers should branch off of the ``develop`` branch, not
-``master``. ``develop`` will be merged into master once it is time to make a
```

Comment:
The second reference to master should probably get back-ticked.

Out of curiosity, do we link to Git Flow http://nvie.com/posts/a-successful-git-branching-model/ or Github Flow https://guides.github.com/introduction/flow/ anywhere?